lib/util.js

```diff
@@ -18,11 +20,7 @@ var PLUGIN_NAME_PREFIX = "eslint-plugin-";
  * @param {Object} source object
  * @returns {void}
  */
-exports.mixin = function(target, source) {
```

Comment:
The above comment says `Public Interface`, but I couldn't find this in the docs. Is it documented somewhere? Otherwise it could probably be removed?